### PR TITLE
chore(doc): proxy note

### PR DESCRIPTION
**Pull request by amolero-nr**
Brief proxy support note in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ Sets timeout for installation task. Overrides the default timeout of 600s.
 
 Verbosity options for the installation (`debug` or `trace`). Writes verbose output to a log file on the host.
 
+#### `https_proxy` (Optional)
```
Comment (Julien4218) [inline README.md:71]: @amolero-nr I thought we should add this to the environment variable section up above, Line 45 I believe

Comment (amolero-nr) [reply README.md]: Thanks again @Julien4218! Moved it up, see here: 84a087655c05d2ca7d3f3f3a329e2f68dd457c8d

File: README.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ Values are read from environment in [vars/main.yml](vars/main.yml)
 - `NEW_RELIC_ACCOUNT_ID`
 - `NEW_RELIC_REGION`
 
+Additionally, an optional `https_proxy` variable can be set to enable a proxy for your installation. Add it to the `environment` keyword in your `playbook`. See [ansible's remote environment](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_environment.html) for more info.
```
Comment (Julien4218) [inline README.md:53]: Just curious, isn't the env variable case sensitive? Shouldn't it be HTTPS_PROXY to work?
@amolero-nr

Comment (rthorn-nr) [reply README.md]: My fault @Julien4218, it is indeed case sensitive

**Review by rthorn-nr — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (amolero-nr) [reply README.md]: Thank you @rthorn-nr!

**Review by Julien4218 — APPROVED** [review]

**Pull request merged**